### DISCUSSION TITLE
Monitor chat updates where i am not a member (not joined this chat)

### DISCUSSION THREAD

**Discussion by allserialtv** [tdlight-team/tdlight-java#198] [Q&A]
Hello.

Is there a way to subscribe to chat updates in my userbot is not a member of this chat. Basically i need to monitor updates of chats where I am not a member of.

Php madelineproto has such option, may be your lib also could provide it

**Comment by cavallium** [comment]
https://tdlight-team.github.io/tdlight-docs/tdlight.api/it/tdlight/jni/TdApi.OpenChat.html
